src/styles.css

```
@@ -0,0 +1,15 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities {
+    input.hide-input-arrows::-webkit-outer-spin-button,
+    input.hide-input-arrows::-webkit-inner-spin-button {
+        -webkit-appearance: none;
+        margin: 0;
+    }
+
+    input[type="number"].hide-input-arrows {
```

```suggestion
    input.tw-hide-input-arrows::-webkit-outer-spin-button,
    input.tw-hide-input-arrows::-webkit-inner-spin-button {
        -webkit-appearance: none;
        margin: 0;
    }

    input[type="number"].tw-hide-input-arrows {
```

What do you think about adding the prefix? 
[Btw no automatic way to do it](https://github.com/tailwindlabs/tailwindcss/discussions/2128)

---

Done 👌